swiftwinrt/Resources/Support/Array+toVector.swift

```diff
@@ -64,4 +64,18 @@ extension ArrayVectorView where T: Equatable {
 
 extension ArrayVectorView {
     public func queryInterface(_ iid: SUPPORT_MODULE.IID) -> IUnknownRef? { nil }
-}
\ No newline at end of file
+}
+
+class ArrayIterator<T>: IIterator {
+    typealias Element = T
+    private var storage: Array<T>
+    private var index: Int = 0
+    init(_ storage: Array<T>){
```

Comment:
Array<T> is a struct, so it is copied to the iterator. Are we okay with that?

Reply:
Arrays are copy-on-write in Swift so there, so no copying would happen in the default case.

Reply:
No issue then